src/frontend/components/PlaylistPortability/index.spec.tsx

```diff
@@ -0,0 +1,191 @@
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import fetchMock from 'fetch-mock';
+import React from 'react';
+
+import { PlaylistPortability } from '.';
+import {
+  playlistLiteMockFactory,
+  playlistMockFactory,
+  videoMockFactory,
+} from '../../utils/tests/factories';
+import { wrapInIntlProvider } from '../../utils/tests/intl';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { Deferred } from '../../utils/tests/Deferred';
+import { wrapInRouter } from '../../utils/tests/router';
+import { Grommet } from 'grommet';
+import MatchMediaMock from 'jest-matchmedia-mock';
+import MatchMedia from 'jest-matchmedia-mock';
+import { Toaster } from 'react-hot-toast';
+
+let matchMedia: MatchMedia;
```

Comment:
I don't understand how you are using this lib. You instantiate it and then it is never used in your tests

Reply:
As I remember, matchmedia is used by react-hot-toast, and matchmedia doesn't exist with jsdom.
https://fun-dev.slack.com/archives/C01GVDYHTR7/p1626086938013400?thread_ts=1626082319.009600&cid=C01GVDYHTR7